packages/cli/src/cli/cogify/action.batch.ts

```diff
@@ -71,7 +73,8 @@ export function createImageryRecordFromJob(job: CogJob): TileMetadataImageryReco
         projection,
         year: extractYearFromName(job.name),
         resolution: extractResolutionFromName(job.name),
-        quadKeys: job.quadkeys,
+        bounds: job.bounds,
+        cogBounds: job.cogBounds,
```

Comment:
I dont like `cogBounds` here, This doesnt actually have to be a COG, it could be any tiff file, it also is not just the `bounds`

could we have maybe `files` ?